content/pandas/concepts/dataframe/terms/isna/isna.md

```diff
@@ -0,0 +1,149 @@
+---
+Title: '.isna()'
+Description: 'It checks whether the objects of a dataframe or a series contain missing or null values and returns a corresponding boolean `True` or `False` value.'
+Subjects:
+    - 'Computer Science'
+    - 'Pandas'
+Tags:
+    - 'Methods'
+    - 'Data Science'
+CatalogContent:
+    - 'learn-python-3'
+    - 'paths/data-science'
+---
+
+
+The `.isna()` method checks whether the objects of a dataframe or a series contain missing or null values `(NA, NaN)` and returns a new object with the same shape as the original object but with boolean values `True` or `False` as the elements. True indicates the presence of null or missing values and False indicates otherwise. The original `DataFrame` object, used to call the method, remains unchanged.'
+
+## Syntax
+
+```py
+# Check for NA values.
+df.isna()
+
+# Look for only boolean type 'True' on the index/row axis.
+df.isna(axis = 0, bool_only = True)
```

Comment:
Are we sure about these parameters? They generate errors for me and I can't find them in the official Pandas documentation: https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.isna.html#pandas.DataFrame.isna